notebooks/outlier-treatment.ipynb

```diff
@@ -0,0 +1,516 @@
+{
```

Comment:
<p>Very clear description. You could insert the stats. score reference link like&nbsp;<a href="https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.zscore.html" target="_blank">this</a>.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neo-empresarial/house_pricing_challenge/pull/44/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Good idea!